cleanlab_studio/studio/trustworthy_language_model.py

```diff
@@ -50,9 +50,47 @@ def __init__(
         timeout: Optional[float] = None,
         verbose: Optional[bool] = None,
     ) -> None:
-        """Use `Studio.TLM()` instead of this method to initialize a TLM.
+        """
```

Comment:
should we actually keep the lazydocs for this __init__() method?  So move the description of the initialization args to the top-level TLM class, and just make it clear there these are the args for `Studio.TLM()`. 

WDYT?  I'm worried it'd be confusing/ugly to show the __init__() method that isn't supposed to be used.

Reply:
that makes sense. updated.